src/ethereum-json-rpc-client/tests/reqwest/rpc_client.rs

```diff
@@ -78,13 +77,13 @@ impl Client for PublicRpcReqwestClient {
                 let result = client.send_rpc_request(request.clone()).await;
 
                 match result {
-                    Ok(Response::Single(Output::Success(_))) => return result,
-                    Ok(Response::Batch(batch))
+                    Ok(RpcResponse::Single(Response::Success(_))) => return result,
+                    Ok(RpcResponse::Batch(batch))
                         if batch
                             .iter()
-                            .all(|output| matches!(output, Output::Success(_))) =>
+                            .all(|output| matches!(output, Response::Success(_))) =>
                     {
-                        return Ok(Response::Batch(batch))
+                        return Ok(RpcResponse::Batch(batch))
                     }
                     Ok(result) => {
                         println!("call failed: {result:?}");
```

Comment:
to be removed. Please remove also the println at line 93 and check if there are others to be removed